src/components/Account.js

```diff
@@ -187,88 +187,84 @@ function Account(props) {
   ));
 
   return (
-    <>
-      <Grid container className={classes.box}>
-        <Grid item xs={3}>
-          <Paper elevation={3} className={classes.menu}>
-            <Menu variant="plain" />
-          </Paper>
-        </Grid>
-        <Grid item xs={9}>
-          <Grid container className={classes.rightBox}>
-            <Grid item xs={12}>
-              <Grid container className={classes.titleBox}>
-                <Grid item>
-                  <AccountIcon className={classes.accountIcon} />
-                </Grid>
-                <Grid item>
-                  <Typography variant="h2">Account</Typography>
-                </Grid>
+    <div style={{ display: 'flex', flexDirection: 'row', flexWrap: 'nowrap' }}>
+      <Paper elevation={3} style={{ width: '300px' }}>
+        <Menu variant="plain" />
+      </Paper>
+
+      <Grid item xs={9} style={{ width: '1000px' }}>
```

Comment:
Don't use a fixed width here.
Instead, set `flexGrow: 1` and remove the `xs={9}` property.
That, combined with the 100% width of the parent element will ensure the content fills the container.